src/clients/ClientDetails.tsx

```diff
@@ -600,6 +601,14 @@ export default function ClientDetails() {
                       reset={() => setupForm(client)}
                     />
                   </Tab>
+                  <Tab
+                    id="Export"
```

Comment:
Any reason this `id` is starting with a capital? Never seen that before.
```suggestion
                    id="export"
```

Reply:
Nice catch, thanks - fixed!